### PR TITLE
Kc create post

### PR DESCRIPTION
**Pull request by kmchandler**
created a method to create new posts. tested successfully in postman. also found an error in do_POST when testing. needed to plug in self.path into resource, _ = self.parse_url(self.path). 

TO TEST: make sure your server is running
create a POST request in postman. add the following to the raw body 
{
    "id": 14,
    "user_id": 2,
    "category_id": 2,
    "title": "test title",
    "publication_date": "2020-10-2",
    "image_url": "https://picsum.photos/200/300",
    "content": "test post content",
    "approved": "approved"
}
send, and the object should come back with a 201 status

### CODE REVIEW

**Review by Massacre-Saint — APPROVED** [review]
This branch created the object! Good job!

**Review by Massacre-Saint — APPROVED** [review]

File: views/posts_requests.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def create_post(new_post):
             ( id, user_id, category_id, title, publication_date, image_url, content, approved )
```
Comment (scamp925) [inline views/posts_requests.py:71]: Need to remove "id" from this line, please :)

File: views/posts_requests.py [hunk 2]
```diff
@@ -71,7 +71,7 @@ def create_post(new_post):
...
         VALUES
             ( ?, ?, ?, ?, ?, ?, ?, ?);
```
Comment (scamp925) [inline views/posts_requests.py:73]: Remove a "?", please

**Review by scamp925 — APPROVED** [review]
Looks awesome!!! Killer work here!

**Pull request merged**